test/parallel/test-cluster-fork-stdio.js

```diff
@@ -28,6 +28,7 @@ if (cluster.isMaster) {
       worker.disconnect();
     }
   });
+  // TODO: stdio[4] is not writable?
   channel.write(buf);
```

Comment:
@cjihrig @addaleax I'm unsure how this is supposed to work? `stdio[4]` is set as `!writable`? I think there is a bug here which this PR hits.